blivet/util.py

```diff
@@ -280,17 +280,17 @@ def get_mount_paths(dev):
 def get_mount_device(mountpoint):
     """ Given a mountpoint, return the device node path mounted there. """
     mountpoint = os.path.realpath(mountpoint)  # eliminate symlinks
-    mounts = open("/proc/mounts").readlines()
     mount_device = None
-    for mnt in mounts:
-        try:
-            (device, path, _rest) = mnt.split(None, 2)
-        except ValueError:
-            continue
+    with open("/proc/mounts") as mounts:
+        for mnt in mounts.readline():
+            try:
+                (device, path, _rest) = mnt.split(None, 2)
+            except ValueError:
+                continue
 
-        if path == mountpoint:
-            mount_device = device
-            break
+            if path == mountpoint:
+                mount_device = device
+                break
```

Comment:
I would expect this fix to be a separate commit.

Reply:
This isn't a separate change, I am just using the `open` contextmanager here so I needed to indent this block. I guess it's just git diff showing it in a weird way.